BE/src/main/java/kr/codesquad/issuetracker/domain/issue/Issue.java

```diff
@@ -0,0 +1,67 @@
+package kr.codesquad.issuetracker.domain.issue;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import kr.codesquad.issuetracker.domain.BaseTimeEntity;
+import kr.codesquad.issuetracker.domain.comment.Comment;
+import kr.codesquad.issuetracker.domain.label.Label;
+import kr.codesquad.issuetracker.domain.member.Member;
+import kr.codesquad.issuetracker.domain.milestone.Milestone;
+
+@Entity
+@Table(name = "issue")
+public class Issue extends BaseTimeEntity {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "issue_id")
+	private Long id;
+
+	private String title;
+
+	@Enumerated(EnumType.STRING)
+	private Status status;
+
+	private String content;
+
+	@ManyToOne(fetch = FetchType.LAZY)
+	@JoinColumn(name = "member_id")
+	private Member member;
+
+	@OneToMany(mappedBy = "issue")
+	private List<Comment> comments = new ArrayList<>();
+
+	@ManyToOne(fetch = FetchType.LAZY)
+	@JoinColumn(name = "milestone_id")
+	private Milestone milestone;
+
+	@ManyToMany
+	@JoinTable(
+		name = "assignee",
+		joinColumns = @JoinColumn(name = "issue_id"),
+		inverseJoinColumns = @JoinColumn(name = "member_id")
+	)
+	private List<Member> members = new ArrayList<>();
```

Comment:
JoinTable을 사용 하는 경우 불필요한 연결테이블 엔티티를 생략 할 수 있습니다 👍

Reply:
member 대신 assignee라는걸 나타내면 어떨까요?

```suggestion
	@ManyToMany
	@JoinTable(
		name = "assignee",
		joinColumns = @JoinColumn(name = "issue_id"),
		inverseJoinColumns = @JoinColumn(name = "member_id")
	)
	private List<Member> assignees = new ArrayList<>();
```

Reply:
이 변수명도 수정하도록 하겠습니다 감사합니다 :)